unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -1797,6 +1797,29 @@ createAuthor =
         _ -> Left $ showPatternHelp createAuthor
     )
 
+gist :: InputPattern
+gist =
+  InputPattern
+    "gist"
+    []
+    [(Required, gitUrlArg)]
+    ( P.lines
+        [ "Publish the current namespace.",
+          "",
+          P.wrapColumn2
+            [ ( "`gist remote`",
+                "publishes the contents of the current namespace into the repo `remote`."
```

Comment:
Sorry, how does this work? I thought gists were user-based not repo-based. Might be worth clarifying this more in the docs too.